src/applications/vaos/tests/e2e/flows/community-care.cypress.spec.js

```diff
@@ -270,13 +267,15 @@ describe('VAOS community care flow using VAOS service', () => {
     // Request detail page should display the same information sent to create the
     // appointment.
     cy.url().should('include', '/requests/mock1');
-    cy.wait('@v2:get:appointment');
-    cy.findByText('Pending primary care appointment');
-    cy.findByText('Your appointment request has been submitted.');
-    cy.findByText('This is a very good reason.');
-    cy.findByText('veteran@gmail.com');
-    // cy.findByText('503-555-1234');
-    cy.findByText('Call morning or evening');
+    cy.wait('@v2:get:appointment', { timeout: 90000 }).then(() => {
+      cy.findByText('Pending primary care appointment', { timeout: 90000 });
+      cy.findByText('Your appointment request has been submitted.');
+      cy.findByText('This is a very good reason.');
+      cy.findByText('veteran@gmail.com');
+      // cy.findByText('503-555-1234');
+      cy.findByText('Call morning or evening');
+    });
+
```

Comment:
Attempt to fix e2e stress test failure by waiting 90 seconds for the api call to finish.